package.json

```diff
@@ -46,6 +50,7 @@
     "test": "npm-run-all test:*",
     "test:karma": "karma start",
     "test:lint": "eslint .",
-    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec"
+    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec",
+    "clientize": "webpack"
```

Comment:
That's right.  The bundled file should never be tracked by source control, it must be run manually by someone who wants to build and deploy it to a given target.